mlrun/datastore/targets.py

```diff
@@ -453,7 +453,10 @@ def _get_store_and_path(self):
             credentials_prefix_secrets,
         )
         if self.get_target_path() and self.get_target_path().startswith("ds://"):
-            return store, store.url + resolved_store_path
+            if hasattr(store, "kind") and store.kind == "v3io":
+                return store, "v3io:/" + resolved_store_path
```

Comment:
It's not really ideal to have a special case for v3io store here in this generic code. Is this because its url is defined with two slashes and here we want it with just a single slash?

Reply:
No - this is because the resolved store.url points to https://iguazio.server. 
I think we should wait for your https://github.com/mlrun/mlrun/pull/5186 to be merged
We will merge this PR then